Gemfile

```diff
@@ -6,3 +6,9 @@ git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 
 gem 'activesupport'
 gem 'octokit'
+gem 'simplecov'
```

Comment:
Simplecov should be added if running tests locally, but isn't necessary for getting decent running.  The other gems are for CI.